src/pydata_sphinx_theme/toctree.py

```diff
@@ -137,18 +137,22 @@ def generate_header_nav_html(n_links_before_dropdown: int = 5) -> str:
         out = "\n".join(links_solo)
 
         # Wrap the final few header items in a "more" dropdown
-        links_dropdown = links_html[n_links_before_dropdown:]
+        links_dropdown = [
+            html.replace("nav-link", "nav-link dropdown-item")
```

Comment:
In order to leverage Bootstrap's [dropdown keyboard accessibility](https://getbootstrap.com/docs/5.2/components/dropdowns/#accessibility), the `dropdown-item` class has to be added to each `a` element in the dropdown menu. This is exactly the same change that #1411 to improve the version switcher.